server/src/ClientToDBAPI/ContactedPeople/mainRoute.ts

```diff
@@ -143,50 +88,36 @@ ContactedPeopleRoute.post('/interactedContacts', (request: Request, response: Re
     const isThereDoneContact = request.body.unSavedContacts.contacts.some((contact : InteractedContact) => contact.contactStatus === DONE_CONTACT);
     const isSingleContact = request.body.unSavedContacts?.contacts.length === 1;
     const workflow = `Saving ${isSingleContact ? 'single': 'all'} contact${isSingleContact ? '' : 's'}`;
-    logger.info({
-        service: Service.SERVER,
-        severity: Severity.LOW,
+    const interactedContactsLogger = logger.setup({
         workflow: `Saving ${isSingleContact ? 'single': 'all'} contact${isSingleContact ? '' : 's'}`,
-        step: `launching graphql API request with parameter: ${JSON.stringify(request.body)}`,
-        user: response.locals.user.id,investigation: response.locals.epidemiologynumber
+        investigation: response.locals.epidemiologynumber,
+        user: response.locals.user.id
     });
+    interactedContactsLogger.info(`launching graphql API request with parameter: ${JSON.stringify(request.body)}`, Severity.LOW);
     graphqlRequest(UPDATE_LIST_OF_CONTACTS, response.locals, { unSavedContacts: JSON.stringify(request.body)})
         .then((result: any) => {
             if(result.data.updateContactPersons) {
-                logger.info({
-                    service: Service.SERVER,
-                    severity: Severity.LOW,
-                    workflow,
-                    step: 'got response from DB',
-                    user: response.locals.user.id,investigation: epidemiologyNumber
-                });
+                interactedContactsLogger.info('got response from DB', Severity.LOW);
                 isThereDoneContact && sendSavedInvestigationToIntegration(epidemiologyNumber, workflow, response.locals.user.id);
                 response.send(result);
             } else if(result.errors) {
                 handleDBErrors(response, result.errors[0].message, workflow);
             }
         })
         .catch(error => {
-            logger.error({
-                service: Service.SERVER,
-                severity: Severity.LOW,
-                workflow,
-                step: `got error from DB from graphql API: ${error}`,
-                user: response.locals.user.id,investigation: response.locals.epidemiologynumber
-            });
+            interactedContactsLogger.error(`got error from DB from graphql API: ${error}`, Severity.HIGH);
```

Comment:
Did you mean Severity.Low?

Reply:
I doubt the original Severity.LOW was correct, considering that in every other catch occasion we use Severity.HIGH. It's probably an mistake @nevobonneh fixed.